### PR TITLE
homie v5 proposal

### PR DESCRIPTION
**Pull request by Tieske**
Replaces #235. Same content for now, but this is a branch local in this repo. Whereas the other PR was in my private repo, which meant that all PR's against it would be in my repo as well, and hence have very low visibility.

The main change is: single topic description based on JSON. More changes are required. This branch/PR is intended as the working version for V5.

### CODE REVIEW

**Review by schaze — APPROVED** [review]

**Review by schaze — APPROVED** [review]

**Review by schaze — APPROVED** [review]

**Review by schaze — APPROVED** [review]

**Comment by schaze** [comment]
Was this waiting for another approval? If yes then I missed it sry...

**Comment by Tieske** [comment]
My idea was to keep this branch as the working version for `V5`. I think we should not merge it into `develop` until this reaches a `beta` or `alpha` level status.

Mostly because this is a major version bump.

**Comment by Tieske** [comment]
Converted it to `draft` again.

**Review by schaze — APPROVED** [review]

**Pull request merged**
